grid2op/tests/test_score_idf_2023.py

```diff
@@ -49,7 +51,9 @@ def act(self, observation: BaseObservation, reward: float, done: bool = False) -
 class TestScoreL2RPN2023(unittest.TestCase):
     
     def setUp(self) -> None:
-        env_name = "l2rpn_case14_sandbox"
+        env_name = os.path.join(
+            PATH_DATA_TEST, "l2rpn_idf_2023_with_alert"
```

Comment:
please change it to "l2rpn_idf_2023"

Reply:
Ok I do so.
See that I need to comment data_feeding_kwargs as I get this error with l2rpn_idf_2023

Grid2op.Exceptions.handlers_exceptions.HandlerError: Grid2OpException ChronicsError HandlerError "No files are found in directory "/home/marotant/dev/Grid2Op_c400c90f/grid2op/data/l2rpn_idf_2023/chronics/2035-01-15_0/prod_vNone". If you don't want to load any chronics, use  "DoNothingHandler" (`from grid2op.Chronics.handlers import DoNothingHandler`) and not "<class 'grid2op.Chronics.handlers.csvHandler.CSVHandler'>" to load chronics."